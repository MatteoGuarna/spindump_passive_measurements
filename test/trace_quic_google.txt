Taken with "tcpdump -n udp -w capture.pcap" on 2019-03-24 on Ubuntu 16.04.
Using Google Chrome Version 73.0.3683.86 (Official Build) (64-bit) (at that time the stable version at deb [arch=amd64] http://dl.google.com/linux/chrome/deb/ stable main)
In Chrome I started 2 youtube videos and did a Google Search.
It uses GQUIC version Q043.
