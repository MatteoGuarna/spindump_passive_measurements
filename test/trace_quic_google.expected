DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113177081 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.7 33626:53 at 1553417113177181 new starting packets 0 0 bytes 0 0
UDP 10.0.2.15 <-> 239.255.255.250 44981:1900 at 1553417113178852 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113186554 measurement starting right 9473 packets 1 0 bytes 75 0 bandwidth 75 0
DNS 10.0.2.15 <-> 31.130.229.7 33626:53 at 1553417113186691 measurement starting right 9510 packets 1 0 bytes 75 0 bandwidth 75 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113249847 measurement closed right 2230 packets 2 1 bytes 140 346 bandwidth 140 346
QUIC 10.0.2.15 <-> 216.58.201.109 3da986ac1023e506-00 (45395:443) at 1553417113250304 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.109 3da986ac1023e506-3da986ac1023e506 (45395:443) at 1553417113267912 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.109 3da986ac1023e506-3da986ac1023e506 (45395:443) at 1553417113267912 measurement up right 17608 packets 1 0 bytes 1378 0 bandwidth 1378 0
UDP 10.0.2.15 <-> 224.0.0.251 5353:5353 at 1553417113446438 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113470294 measurement closed right 1881 packets 3 2 bytes 201 675 bandwidth 201 675
QUIC 10.0.2.15 <-> 172.217.23.206 0785522d9d886745-00 (56696:443) at 1553417113503066 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 172.217.23.206 0785522d9d886745-0785522d9d886745 (56696:443) at 1553417113521124 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.23.206 0785522d9d886745-0785522d9d886745 (56696:443) at 1553417113521124 measurement up right 18058 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113618731 measurement closed right 2739 packets 4 3 bytes 264 818 bandwidth 264 818
QUIC 10.0.2.15 <-> 216.58.201.67 0afdf4f1c557ec4d-00 (50407:443) at 1553417113619285 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417113627386 measurement closed right 2253 packets 5 4 bytes 323 933 bandwidth 323 933
QUIC 10.0.2.15 <-> 216.58.201.67 0afdf4f1c557ec4d-48652a4546496614 (50407:443) at 1553417113629125 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.67 00-0afdf4f1c557ec4d (50407:443) at 1553417113637577 change starting packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 10.0.2.15 <-> 216.58.201.67 00-0afdf4f1c557ec4d (50407:443) at 1553417113637577 measurement up right 18292 packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 172.217.23.193 <-> 10.0.2.15 48652a4546496614-00 (443:50108) at 1553417113647174 new starting packets 0 0 bytes 0 0
QUIC 172.217.23.193 <-> 10.0.2.15 48652a4546496614-48652a4546496614 (443:50108) at 1553417113659432 change starting packets 2 0 bytes 2756 0 bandwidth 2756 0
QUIC 172.217.23.193 <-> 10.0.2.15 48652a4546496614-48652a4546496614 (443:50108) at 1553417113659432 measurement up right 11873 packets 2 0 bytes 2756 0 bandwidth 2756 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417114014281 measurement closed right 2072 packets 6 5 bytes 396 1308 bandwidth 396 1308
QUIC 10.0.2.15 <-> 172.217.23.226 6219ee777f9f2312-00 (33856:443) at 1553417114014564 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 172.217.23.226 6219ee777f9f2312-6219ee777f9f2312 (33856:443) at 1553417114035926 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.23.226 6219ee777f9f2312-6219ee777f9f2312 (33856:443) at 1553417114035926 measurement up right 21362 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417114850647 measurement closed right 2779 packets 8 6 bytes 515 1422 bandwidth 396 1422
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417114850791 measurement closed right 2434 packets 8 7 bytes 515 1755 bandwidth 396 1422
QUIC 10.0.2.15 <-> 216.58.201.99 949482ee71fef483-00 (38162:443) at 1553417114850977 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.99 949482ee71fef483-949482ee71fef483 (38162:443) at 1553417114870091 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.99 949482ee71fef483-949482ee71fef483 (38162:443) at 1553417114870091 measurement up right 19114 packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.67 6a892e9705db1411-00 (47601:443) at 1553417117203294 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417117207230 measurement closed right 2418 packets 11 8 bytes 695 2079 bandwidth 119 1422
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417117208073 measurement closed right 1880 packets 11 9 bytes 695 2225 bandwidth 119 657
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417117208400 measurement closed right 2023 packets 11 10 bytes 695 2343 bandwidth 119 657
QUIC 10.0.2.15 <-> 216.58.201.67 6a892e9705db1411-6a892e9705db1411 (47601:443) at 1553417117221392 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.67 6a892e9705db1411-6a892e9705db1411 (47601:443) at 1553417117221392 measurement up right 18098 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417117287327 measurement closed right 12067 packets 12 11 bytes 773 2448 bandwidth 119 657
QUIC 10.0.2.15 <-> 74.125.11.27 9c2e937baef6b800-00 (33166:443) at 1553417117288003 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 74.125.11.27 9c2e937baef6b800-9c2e937baef6b800 (33166:443) at 1553417117299895 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 74.125.11.27 9c2e937baef6b800-9c2e937baef6b800 (33166:443) at 1553417117299895 measurement up right 11892 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417118050343 measurement closed right 4512 packets 14 12 bytes 898 2826 bandwidth 119 657
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417118050498 measurement closed right 4608 packets 14 13 bytes 898 3154 bandwidth 119 657
QUIC 10.0.2.15 <-> 216.58.201.110 d652d0451aa7e2c7-00 (45409:443) at 1553417118058931 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.110 d652d0451aa7e2c7-09e13943a6defda3 (45409:443) at 1553417118059480 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.110 00-d652d0451aa7e2c7 (45409:443) at 1553417118080290 change starting packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 10.0.2.15 <-> 216.58.201.110 00-d652d0451aa7e2c7 (45409:443) at 1553417118080290 measurement up right 21359 packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 172.217.23.195 <-> 10.0.2.15 09e13943a6defda3-00 (443:32822) at 1553417118084859 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417118084945 measurement closed right 15407 packets 17 14 bytes 1107 3486 bandwidth 119 657
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417118085136 measurement closed right 15292 packets 17 15 bytes 1107 3818 bandwidth 119 657
QUIC 172.217.23.195 <-> 10.0.2.15 09e13943a6defda3-ea7761ae4ab94abe (443:32822) at 1553417118087999 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 172.217.23.195 <-> 10.0.2.15 00-09e13943a6defda3 (443:32822) at 1553417118108358 change starting packets 2 1 bytes 2756 1378 bandwidth 2756 1378
QUIC 172.217.23.195 <-> 10.0.2.15 00-09e13943a6defda3 (443:32822) at 1553417118108358 measurement up right 8516 packets 2 1 bytes 2756 1378 bandwidth 2756 1378
QUIC 10.0.2.15 <-> 216.58.201.99 ea7761ae4ab94abe-00 (59603:443) at 1553417118131438 new starting packets 0 0 bytes 0 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417118133045 measurement closed right 61644 packets 17 16 bytes 1107 4161 bandwidth 119 657
QUIC 10.0.2.15 <-> 216.58.201.99 ea7761ae4ab94abe-ea7761ae4ab94abe (59603:443) at 1553417118147243 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.99 ea7761ae4ab94abe-ea7761ae4ab94abe (59603:443) at 1553417118147243 measurement up right 15805 packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.23.227 82587255616d9b66-00 (56170:443) at 1553417118513398 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 172.217.23.227 82587255616d9b66-1ce187333973f0ab (56170:443) at 1553417118514131 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.23.227 00-82587255616d9b66 (56170:443) at 1553417118531387 change starting packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 10.0.2.15 <-> 172.217.23.227 00-82587255616d9b66 (56170:443) at 1553417118531387 measurement up right 17989 packets 1 1 bytes 1378 1378 bandwidth 1378 1378
QUIC 172.217.23.226 <-> 10.0.2.15 1ce187333973f0ab-00 (443:41539) at 1553417118532605 new starting packets 0 0 bytes 0 0
QUIC 172.217.23.226 <-> 10.0.2.15 1ce187333973f0ab-1ce187333973f0ab (443:41539) at 1553417118537928 change starting packets 2 0 bytes 2756 0 bandwidth 2756 0
QUIC 172.217.23.226 <-> 10.0.2.15 1ce187333973f0ab-1ce187333973f0ab (443:41539) at 1553417118537928 measurement up right 5312 packets 2 0 bytes 2756 0 bandwidth 2756 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417120171172 measurement closed right 5982 packets 20 17 bytes 1275 4508 bandwidth 592 657
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417120171952 measurement closed right 6489 packets 20 19 bytes 1275 4772 bandwidth 592 2429
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417121677599 measurement closed right 2533 packets 21 20 bytes 1334 4901 bandwidth 168 2429
QUIC 10.0.2.15 <-> 172.217.18.163 c44b3575f1e85cb3-00 (53181:443) at 1553417121677968 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 172.217.18.163 c44b3575f1e85cb3-c44b3575f1e85cb3 (53181:443) at 1553417121695594 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.18.163 c44b3575f1e85cb3-c44b3575f1e85cb3 (53181:443) at 1553417121695594 measurement up right 17626 packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.78 b3917c3aa5cf8c3f-00 (58931:443) at 1553417121849552 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.78 b3917c3aa5cf8c3f-b3917c3aa5cf8c3f (58931:443) at 1553417121869677 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.78 b3917c3aa5cf8c3f-b3917c3aa5cf8c3f (58931:443) at 1553417121869677 measurement up right 20125 packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.118 6b1ce273cc10c84b-00 (53038:443) at 1553417122573755 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.118 6b1ce273cc10c84b-6b1ce273cc10c84b (53038:443) at 1553417122594002 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.118 6b1ce273cc10c84b-6b1ce273cc10c84b (53038:443) at 1553417122594002 measurement up right 20247 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417122981269 measurement closed right 2610 packets 22 21 bytes 1407 4995 bandwidth 59 393
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417123236762 measurement closed right 24786 packets 23 22 bytes 1475 5339 bandwidth 59 94
QUIC 10.0.2.15 <-> 172.217.23.230 9b9ccb4b2f200a6d-00 (40999:443) at 1553417123237965 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 172.217.23.230 9b9ccb4b2f200a6d-9b9ccb4b2f200a6d (40999:443) at 1553417123256650 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 172.217.23.230 9b9ccb4b2f200a6d-9b9ccb4b2f200a6d (40999:443) at 1553417123256650 measurement up right 18685 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417123812719 measurement closed right 43745 packets 24 23 bytes 1533 5472 bandwidth 59 94
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417124564595 measurement closed right 17802 packets 25 24 bytes 1598 5623 bandwidth 199 94
QUIC 10.0.2.15 <-> 216.58.201.78 18b87731fc7d93f7-00 (40952:443) at 1553417124565196 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 216.58.201.78 18b87731fc7d93f7-18b87731fc7d93f7 (40952:443) at 1553417124595930 change starting packets 1 0 bytes 1378 0 bandwidth 1378 0
QUIC 10.0.2.15 <-> 216.58.201.78 18b87731fc7d93f7-18b87731fc7d93f7 (40952:443) at 1553417124595930 measurement up right 30734 packets 1 0 bytes 1378 0 bandwidth 1378 0
DNS 10.0.2.15 <-> 31.130.229.6 33626:53 at 1553417131343492 measurement closed right 46202 packets 26 25 bytes 1676 5728 bandwidth 65 628
QUIC 10.0.2.15 <-> 173.194.187.134 202b98a9db68b8bb-00 (48731:443) at 1553417131348437 new starting packets 0 0 bytes 0 0
QUIC 10.0.2.15 <-> 173.194.187.134 202b98a9db68b8bb-202b98a9db68b8bb (48731:443) at 1553417131360065 change starting packets 3 0 bytes 1795 0 bandwidth 1795 0
QUIC 10.0.2.15 <-> 173.194.187.134 202b98a9db68b8bb-202b98a9db68b8bb (48731:443) at 1553417131360065 measurement up right 11380 packets 3 0 bytes 1795 0 bandwidth 1795 0
